fe1-web/__mocks__/publishMock.ts

```diff
@@ -0,0 +1,9 @@
+import { publish } from 'network/JsonRpcApi';
+
+/**
+ * Mocks the publish method that publish messages.
+ */
+jest.mock('network/JsonRpcApi.ts', () => ({
+  publish: jest.fn(() => Promise.resolve()),
```

Comment:
would it make sense to define this in `network/__mocks__/JsonRpcApi.ts` ? 
We could maybe have a (partial) mock re-usable everywhere.